agreements.md

```diff
@@ -176,6 +176,32 @@ This solution is meant to allow the team to auto-calibrate and recognize what co
 If the offensive comment is seen as severe and above the grade of this agreement, it will be raised to the HR circle to decide on further steps.
 Fortunately, so far, there have been no such situations.
 
+## Constructive feedback
+
+We aim to maintain a positive work environment where feedback is given constructively to promote growth and improvement.
```

Comment:
```suggestion
We aim to maintain a positive work environment where feedback is given constructively to promote growth.
```
growth and improvement seem to be exactly same thing in this context

Reply:
growth = personal, improvement = process

if we want to keep one I would keep improvement. @mjurbanski-reef what do you think?